src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Main.cs

```diff
@@ -135,8 +143,11 @@ public List<Result> Query(Query query)
 
                                 return hide;
                             };
+                            r.Score = score;
                             r.ContextData = searchResult;
 
+                            score -= 1;
+

```

Comment:
According to @somil55's comment, we can use `Enumerable.OrderBy` to preserve the order of elements. This way we don't have to have elements with negative score as it seems a bit counter-intuitive.

Reply:
I'll remove the scoring since @somill55 made a PR for changing to stable sorting (#5850 )

Reply:
I've removed the scoring. While testing the PR you may notice the ordering change when using search keywords that require LIKE query (like "abcd.a") since it will first load the CONTAINS result and then the CONTAINS+LIKE result but the ordering won't be maintained till #5850 is merged.